docs/guides/integration/github.md

```diff
@@ -2,165 +2,70 @@
 
 ## Installation
 
-uv installation differs depending on the platform:
+For use with GitHub Actions, we recommend the official
+[`astral-sh/setup-uv`](https://github.com/astral-sh/setup-uv) action, which installs uv, adds it to
+PATH, (optionally) persists the cache, and more, with support for all uv-supported platforms.
 
-=== "Linux"
+To install the latest version of uv:
 
-    ```yaml title="example.yml"
-    name: Example on Linux
-
-    jobs:
-      uv-example-linux:
-        name: python-linux
-        runs-on: ubuntu-latest
-
-        steps:
-          - uses: actions/checkout@v4
-
-          - name: Set up uv
-            # Install latest uv version using the installer
-            run: curl -LsSf https://astral.sh/uv/install.sh | sh
-    ```
-
-=== "macOS"
-
-    ```yaml title="example.yml"
-    name: Example on macOS
-
-    jobs:
-      uv-example-macos:
-        name: python-macos
-        runs-on: macos-latest
-
-        steps:
-          - uses: actions/checkout@v4
-
-          - name: Set up uv
-            # Install latest uv version using the installer
-            run: curl -LsSf https://astral.sh/uv/install.sh | sh
-    ```
-
-=== "Windows"
-
-    ```yaml title="example.yml"
-    name: Example on Windows
+```yaml title="example.yml" hl_lines="14-16"
+name: Example
 
-    jobs:
-      uv-example-windows:
-        name: python-windows
-        runs-on: windows-latest
+jobs:
+  uv-example:
+    name: python
+    runs-on: ubuntu-latest
 
-        steps:
-          - uses: actions/checkout@v4
+    steps:
+      - uses: actions/checkout@v4
 
-          - name: Set up uv
-            # Install latest uv version using the installer
-            run: irm https://astral.sh/uv/install.ps1 | iex
-            shell: powershell
-    ```
+      - name: Install uv
+        uses: astral-sh/setup-uv@v1
+        with:
+          # Install the latest version of uv (default).
```

Comment:
`latest` seems not the default?

> By default this action installs the version defined as default in action.yml. This gets automatically updated in a new release of this action when a new version of uv is released. If you don't want to wait for a new release of this action you can use version: latest.

Reply:
https://github.com/astral-sh/setup-uv/pull/37